packages/fuel-indexer-database/postgres/migrations/20231012190730_indexer_status.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS index_status (
+   indexer_id bigint,
+   namespace varchar(32) not null,
```

Comment:
@lostman If we're keeping track of indexer info in `index_registry` and using a foreign key for `index_status` do we need to also keep the namespace and identifier here as well?